fem/gslib.hpp

```diff
@@ -156,10 +159,12 @@ class FindPointsGSLIB
    virtual void Interpolate(const GridFunction &field_in, Vector &field_out);
    /** Search positions and interpolate */
    void Interpolate(const Vector &point_pos, const GridFunction &field_in,
-                    Vector &field_out);
+                    Vector &field_out,
```

Comment:
Would it also make sense to have a field_out_ordering here too?

Reply:
It's also a little weird to me that field_out is a vector instead of a GridFunction.  If it were a GridFunction it would have an ordering defined inside of it.